inc/Engine/Media/PreloadFonts/Context/Context.php

```diff
@@ -47,4 +90,22 @@ public function is_mobile_allowed(): bool {
 			&& $this->options->get( 'do_caching_mobile_files', 0 )
 			&& wp_is_mobile();
 	}
+
+	/**
+	 * Get array of fonts to be excluded.
+	 *
+	 * @return array
+	 */
+	public function get_exclusions(): array {
+		$lists      = $this->data_manager->get_lists();
+		$lists      = isset( $lists->delay_js_exclusions ) ? $lists->delay_js_exclusions : [];
+		$exclusions = array_merge( $this->exclusions, $lists );
+
+		/**
+		 * Filters excluded fonts.
+
+		 * @param array $exclusions Array of fonts to exclude.
+		 */
+		return wpm_apply_filters_typed( 'array', 'rocket_preload_fonts_excluded_fonts', $exclusions );
```

Comment:
```suggestion
		return wpm_apply_filters_typed( 'string[]', 'rocket_preload_fonts_excluded_fonts', $exclusions );
```
this would be better?

Reply:
~~I don't think `string[]` is a supported type for `wpm_apply_filters_typed`~~ 
Actually it is. my bad.